src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Condition/IsCaptureCompletedPlugin.php

```diff
@@ -24,6 +24,8 @@ class IsCaptureCompletedPlugin extends AbstractPlugin implements ConditionInterf
     public const CAPTURE_TRANSACTION_ACCEPTED = AfterPayConfig::API_TRANSACTION_OUTCOME_ACCEPTED;
 
     /**
+     * {@inheritDoc}
+     *
```

Comment:
Add a specification to a method:
- Checks if the capture operation was successfully accepted for the order.